web/ui/react-app/src/utils/index.ts

```diff
@@ -243,9 +243,26 @@ export const encodePanelOptionsToQueryString = (panels: PanelMeta[]): string =>
   return `?${panels.map(toQueryString).join('&')}`;
 };
 
+export const encodeToQueryString = (expr: string): string => {
+  return `?expr=${expr}`;
+};
+
 export const createExpressionLink = (expr: string): string => {
   return `../graph?g0.expr=${encodeURIComponent(expr)}&g0.tab=1&g0.stacked=0&g0.show_exemplars=0.g0.range_input=1h.`;
 };
+
+export const createExpressionLinkAlert = (expr: string): string => {
```

Comment:
This and the other two functions below seem unused and can be deleted?

Reply:
ready! ;)